.github/workflows/test.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           find ./tests -type d | while IFS= read -r dir
```

Comment:
lets remove all .github file changes from this pr please

Reply:
k